bot/exts/events/advent_of_code/_cog.py

```diff
@@ -217,36 +217,31 @@ async def aoc_day_and_star_leaderboard(
         brief="Get a snapshot of the PyDis private AoC leaderboard",
     )
     @whitelist_override(channels=AOC_WHITELIST_RESTRICTED)
-    async def aoc_leaderboard(
-            self,
-            ctx: commands.Context,
-            *,
-            self_placement_name: Optional[str] = None,
-    ) -> None:
+    async def aoc_leaderboard(self, ctx: commands.Context, *, aoc_name: Optional[str] = None) -> None:
         """
         Get the current top scorers of the Python Discord Leaderboard.
 
-        Additionally you can specify a `self_placement_name`
-        that will append the specified profile's personal stats to the top of the leaderboard
+        Additionally you can specify an `aoc_name` that will append the
+        specified profile's personal stats to the top of the leaderboard
         """
         # Strip quotes from the self placement name if needed (e.g. "My Name" -> My Name)
```

Comment:
`# Additionally keeps the second layer, if a user has a name with quotes on both ends.`